sql/plsql/PlSqlParser.g4

```diff
@@ -176,7 +178,7 @@ procedure_spec
 
 function_spec
     : FUNCTION identifier ('(' parameter ( ',' parameter)* ')')?
-      RETURN type_spec (DETERMINISTIC)? (RESULT_CACHE)? ';'
+      RETURN type_spec (PIPELINED)? (DETERMINISTIC)? (RESULT_CACHE)? ';'
```

Comment:
Please remove excess parentheses:

```g4
RETURN type_spec PIPELINED? DETERMINISTIC? RESULT_CACHE? ';'
```

Reply:
Yes, you're right. Unnecessary bloat brackets :D I'll fix that